src/SimpleVersion.Core/Pipeline/Formatting/Semver2FormatProcess.cs

```diff
@@ -8,7 +8,26 @@ public class Semver2FormatProcess : ICalculatorProcess
         public void Apply(VersionContext context)
         {
             var labelParts = new List<string>(context.Configuration.Label);
-            var metaParts = new List<string>(context.Configuration.MetaData);
+            var metaParts = new List<string>(context.Configuration.MetaData);
+            var AddBranchName = true;
```

Comment:
Hopefully we'll be able to re-purpose some of the logic here.  Perhaps the label parts could be calculated ahead of the format processes so that the parts are calculated once, but each format could pick the parts/string format required.